src/Workleap.DomainEventPropagation.Analyzers/Workleap.DomainEventPropagation.Analyzers.csproj

```diff
@@ -0,0 +1,31 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
I think analyzers project  also need to be included in the abstraction project like we did for Mongo and Mediatr
https://github.com/gsoft-inc/wl-extensions-mongo/blob/main/src/Workleap.Extensions.Mongo.Abstractions/Workleap.Extensions.Mongo.Abstractions.csproj

Reply:
Yes also this will required a small change to the Build.ps1 script, you can compare with the other Mongo & MediatR repos

Reply:
Why is it required, is it for packaging ?